.circleci/config.yml

```diff
@@ -167,14 +191,14 @@ jobs:
       - run:
           name: Build XCFramework
           command: |
-            if [ "${CIRCLE_BRANCH}" == "main" ]; then
+            if [ "${CIRCLE_BRANCH}" == "staging" ] || [ "${CIRCLE_BRANCH}" == "dev" ]; then
```

Comment:
Can you update to run on "main" as well? There can be cases where we merge directly to main so this workflow should get run again.

Reply:
Yes, but after the merge happens the target branch is run. The source branch can come from anywhere, usually its staging to main but it could also be a different branch or a branch in a forked repository. So as a sanity check before a release we should run these checks on main.